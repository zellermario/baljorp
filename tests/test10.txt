create L m1 0
create SJ m2 0
addNeighbor m1 1 m2
addNeighbor m2 2 m1

create Eszkimo e m1
create Buvarruha b player e

useobject b e 1
pass e