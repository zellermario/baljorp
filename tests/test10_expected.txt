Fields:
L m1 (0) - object: x
	neighbours: [(1,m2)]
SJ m2 (0) - object: x
	neighbours: [(2,m1)]

Buildings:

Actors:
> Eszkimo e (heat: 5) on m2
  Inventory: [b]